src/Fixer/FunctionNotation/VoidReturnFixer.php

```diff
@@ -122,6 +118,27 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
         }
     }
 
+    /**
+     * @return array<array{int, string}>
+     */
+    private function getExcludedFunctions()
+    {
+        $excludedNames = ['__construct', '__destruct', '__clone'];
+
+        if (\PHP_VERSION_ID >= 80000) {
```

Comment:
Should these be excluded only in PHP 8+ or always? I don't have a preference.

Reply:
Always IMO. Longer term, maybe we should have a target min/max PHP version, rather than relying so much on the runtime version, so people could specify the min and max versions of PHP they want their code to run on.

Reply:
> Should these be excluded only in PHP 8+ or always? I don't have a preference.

If I understand right.. if someone is working on 7.4, then run the fixer - `__isset` would not get excluded, currently, then the person migrates to 8.0 and have to do a manual code update to remove the `: void` -> is that the case? if so, I would suggest to always exclude those methods